bin/rsq.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# -*- coding: utf-8 -*-
+#
+# Execute Redshift Query.
+#
+# Usage:
+#
+#   bash rsq.sh <query-name> <other-psql-options>
+#
+# Example:
+#
+#   bash rsq.sh auth_funnel -W -v variable1=value1 -v=variable2=value2
+#
+# Explain:
+#
+#   <query-name> is defined in rsq.py file, rsq.py will find the absolute path of a sql file by query name
+#   <other-psql-option> can be any psql options other than -h, -p, -d, -U, -W, -f. Usually you use -v var1=value1 -v var2=value2 ... to pass parameters.
+#
+# This bin tools depends on rsq.py
+#
+# How to test:
+#
+#   bash tests/test_rsq.sh # password = 'password'
+#
+# Expected result:
+#
+#  count
+# -------
+#      2
+
+dir_bin="$( cd "$(dirname "$0")" ; pwd -P )"
+path_rsq_py="${dir_bin}/rsq.py"
```

Comment:
I don't quite understand why you've included both a .py and a .sh - it might be a bit simpler to have it all in one python script?

Reply:
I do can call bash command in python script, but construct bash command as string in python is too hacky and not very straightforward and hard to understand to others who familiar with bash but not familiar with python.

So I choose to use python just to process input, return string, print multi-line help information.